src/ImageSharp.Web/Processors/ResizeWebProcessor.cs

```diff
@@ -374,15 +249,5 @@ private static ushort GetExifOrientation(FormattedImage image, CommandCollection
             image.TryGetExifOrientation(out ushort orientation);
             return orientation;
         }
```

Comment:
I'd rather use an explicit value when the EXIF orientation can't be retrieved from the image:

```
private static ushort GetExifOrientation(FormattedImage image, CommandCollection commands, CommandParser parser, CultureInfo culture)
{
    if ((commands.Contains(Orient) && !parser.ParseValue<bool>(commands.GetValueOrDefault(Orient), culture)) ||
        !image.TryGetExifOrientation(out ushort orientation))
    {
        return ExifOrientationMode.Unknown;
    }

    return orientation;
}
```

This is (currently) functionally the same, so it's more a styling preference.

Reply:
Another minor styling issue I noticed is that the order of the commands differ between the constants and `ResizeCommands`, plus `Sampler` and `PadColor` can be assigned in the `ResizeOptions` object initializer.

Reply:
There’s less codegen in my version